docs/dev-guide/adding-grid-support/adding-grid-support-step-by-step-guide/add-grid-config.md

```diff
@@ -1,3 +1,175 @@
 # Add New Grid Configuration to E3SM
 
+In addition to generating input data to support a new grid, several code modifications are required before E3SM can run with the grid. However, the specific changes will depend on how the grid will be used. The intendend model configuration for the new grid will change which files need to be modified. For instance, a grid intended for aquaplanet experiments does not require as many changes as a historical AMIP-style run.
+
+The guidelines here are meant to outline various possible changes the user should consider when adding support for a new grid. This document cannot be exhaustive, and it is important that the user understands the changes they are making. It is often useful to use a pre-existing grid configuration as a template. Note that the guidelines here are only relevant for "horizontal" grids. Similar considerations are needed to support a new vertical grid.
+
+When setting up a new grid you will need to edit some or all of these files:
+
+- `cime_config/config_grids.xml`
+- `components/eam/bld/config_files/horiz_grid.xml`
+- `components/eam/bld/namelist_files/namelist_defaults_eam.xml`
+- `components/eam/bld/namelist_files/namelist_definition.xml`
+- `components/elm/bld/namelist_files/namelist_definition.xml`
+- `components/elm/bld/namelist_files/namelist_defaults.xml`
+
+## Mono-Grid vs Bi-Grid vs Tri-Grid
+
+The mono-bi-tri grid options in E3SM can be confusing, but it's important to understand what these terms mean when adding a new grid to E3SM. At the surface these terms mean that the whole model either using a single grid for all componennt models, or a combination of 2 or 3 grids shared among the component models. Note that mono-grid and bi-grid terms often ignore that the river model needs to be on i/ts own regular lat-lon grid (for now).
+
+Historically, climate models would use a single grid for all components (i.e. mono-grid), but this is often not the case anymore. In E3SM the ocean and sea-ice components often use targeted regional refinement with special consideration of ocean mesoscale eddies, whereas the atmosphere will generally use a globally homogenous grid. In practice, the main difference between "bi" and "tri" grids often comes down to whether the land surface model shares a grid with the atmosphere or not. The component coupler is in charge of facilitating communication between component models, primarily through fluxes, and so mapping files are needed to support a combination of different grids. E3SMv3 uses a tri-grid configuration for production simulations.
+
+## Grid Naming Conventions
+
+The atmosphere grid name should always indicate the "ne" value and add "pg2" to indicate that the physgrid is being used. For a regionally refined mesh (RRM) the grid name should always start with `ne0` followed a descriptive string that includes the region being refined and the degree of refinement.
```

Comment:
This description doesn't fit with the RRM example shown below. You probably need to introduce the difference between "np4" and "pg2" here to make the RRM names make more sense.

Reply:
updated this paragraph